guides/source/getting_started_with_devcontainer.md

```diff
@@ -60,8 +60,8 @@ take care of installing the correct Ruby and Rails versions for you.
 
 To install `rails-new`, follow the installation instructions [in the README](https://github.com/rails/rails-new?tab=readme-ov-file#installation).
 
-Creating the Blog Application
------------------------------
+Creating the Store Application
```

Comment:
Hi @duleorlovic - this is a tutorial for a completely different app. This one walks you through creating an e-commerce app, hence the `store` name.

Reply:
It would be great to replace these references out to use the store app models/controllers in future, once this PR merged, for consistency across guides. 

Ex this guide refers to blog example, which can feel out of context or in some places we refer to existing knowledge or "Blog related models, setup , etc"

https://github.com/rails/rails/blob/5608e7b5e4e71e39c8dee31253820fe0955ec494/guides/source/debugging_rails_applications.md 

Not in the scope of this PR though.